audit/dockerhost/dockerhost.go

```diff
@@ -73,12 +73,19 @@ func CheckKernelVersion(client *client.Client) audit.Result {
 	}
 
 	constraints, _ := version.NewConstraint(">= 3.10")
-	hostVersion, _ := version.NewVersion(info.KernelVersion)
+	hostVersion, err := version.NewVersion(info.KernelVersion)
+	if err != nil {
+		// necessary fix for Fedora 23
```

Comment:
this should write /necessary fix for incompatible kernel versions (e.g. Fedora 23)